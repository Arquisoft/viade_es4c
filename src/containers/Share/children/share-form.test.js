import React from 'react';
import { shallow } from 'enzyme';
import ShareForm from "./share-form.component";

describe('ShareForm', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<ShareForm debug />);
  
    expect(component).toMatchSnapshot();
  });
});
