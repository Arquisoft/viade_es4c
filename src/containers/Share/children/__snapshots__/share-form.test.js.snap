// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShareForm should render correctly in "debug" mode 1`] = `
<form
  onSubmit={[Function]}
>
  <label>
    Route's webID:
    <input
      name="route"
      onChange={[Function]}
      type="text"
    />
  </label>
  <label>
    Insert your friend's webID:
    <input
      name="friend"
      onChange={[Function]}
      type="text"
    />
  </label>
  <input
    type="submit"
    value="Submit"
  />
</form>
`;
